modules/microshift-oc-mirror-install-catalog-cluster.adoc

```diff
@@ -17,41 +17,53 @@ After you mirror your image set to the mirror registry, you must apply the gener
 
 . Apply the catalog source configuration file from the results directory to create the catalog source object by running the following command:
 +
+[source,terminal,subs="+quotes"]
+----
+$ oc apply -f ./_<v2_workspace>_/working-dir/cluster-resources/catalogSource-cs-redhat-catalog.yaml <1>
+----
+<1> Replace _<v2_workspace>_ with the directory you used to store custom resources for the mirroring process.
++
+.Example output
 [source,terminal]
 ----
-$ oc apply -f ./oc-mirror-workspace/results-1708508014/catalogSource-cs-redhat-operator-index.yaml
+catalogsource.operators.coreos.com/cs-redhat-catalog created
 ----
+
+. For reference, see the following example file:
```

Comment:
For a future PR, consider whether this should stand alone as a step. It sort of seems like it should be part of the step before it.